src/pages/Search.js

```diff
@@ -5,10 +5,10 @@ import { useParams } from 'react-router-dom';
 const Search = () => {
   const { subreddit = 'javascript' } = useParams();
   return (
-    <div>
+    <main id="search">
```

Comment:
What is the id used for? (Adding all these comments to keep track of the usage of ids :wink:)